pkg/deploy/dashboard/dashboard_deployment_test.go

```diff
@@ -14,6 +14,8 @@ package dashboard
 import (
 	"os"
 
+	osConfigV1 "github.com/openshift/api/config/v1"
```

Comment:
```suggestion
	configv1 "github.com/openshift/api/config/v1"
```
let's be consistent in naming